authz/providers/azure/rbac/rbac.go

```diff
@@ -273,11 +293,16 @@ func (a *AccessInfo) CheckAccess(request *authzv1beta1.SubjectAccessReviewSpec)
 	glog.V(7).Infof("checkaccess response: %s, Configured ARM call limit: %d", string(data), a.armCallLimit)
 	if resp.StatusCode != http.StatusOK {
 		glog.Errorf("error in check access response. error code: %d, response: %s", resp.StatusCode, string(data))
-		if resp.StatusCode == http.StatusTooManyRequests {
-			glog.V(10).Infoln("Closing idle TCP connections.")
+		// metrics for calls with StatusCode >= 300 
+		if resp.StatusCode >= http.StatusMultipleChoices {
+                        glog.V(10).Infoln("Closing idle TCP connections.")
```

Comment:
move this inside if loop

Reply:
we are closing the connection outside the statustoomany wala if , hence i put this outside

Reply:
yeah but u  changed outer if condition. it may go inside if due to other error too... we should close connection also inside now only for 429 error.